packages/openfeature-browser-provider/src/index.ts

```diff
@@ -100,66 +100,96 @@ export class BucketBrowserSDKProvider implements Provider {
     await this.initialize(newContext);
   }
 
-  resolveBooleanEvaluation(
+  private resolveFeature<T extends null | boolean | string | number | object>(
```

Comment:
In the interest of getting this PR in, i suggest we stash the OpenFeature changes and discuss them separately.

Reply:
Will move